crates/solidity/testing/perf/benches/iai/main.rs

```diff
@@ -36,9 +36,14 @@ fn query(files: Vec<ParsedFile>) {
     black_box(tests::query::run(&files));
 }
 
+#[library_benchmark]
+fn bindings() {
```

Comment:
does the new `bindings` test intend to measure built-in bindings only?
Do we expect it to differ than normal bindings, so we would want to specifically measure it?
In that case, I suggest:

1. renaming it to `init_bindings()` because the other two are also about "bindings"
2. moving the parsing part to `tests::init_bindings::setup()` and have `tests::init_bindings::run()` only about adding the cst ..

If not, what does it intend to measure?

Reply:
The intention is to measure the cost of constructing the Bindings (ie. parse the rules and parse and add the built-ins). This includes parsing the built-ins source file, so I'm hesitant to move that into the setup. Agree on renaming to `init_bindings()`.